api/app/Policies/UserPolicy.php

```diff
@@ -38,7 +38,7 @@ public function view(User $user, User $model)
                     $user->rolesTeams()->get()->pluck('id')
                 )
             ) ||
-            ($user->isAbleTo('view-team-communityInterest')
+            ($user->isAbleTo('view-team-communityTalent')
```

Comment:
We still need to solve this problem:
> This should probably be checking view-team-communityTalent for the specific community.

I'm imagining something like this, where we check the permission for each community that the user belongs to:
```php
$communitiesTheUserBelongsTo = $user->rolesTeams()....
$userCanViewCommunityTalent = $communitiesTheUserBelongsTo
  ->contains(fn($community) => $user->isAbleTo('view-team-communityTalent', $community))

```

Reply:
This should be addressed in [57af7be](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13130/commits/57af7be4d74024bc0aade93bc29cf135dd5d8c7c).